CodeConverter/CSharp/CommonConversions.cs

```diff
@@ -145,7 +145,7 @@ public record VariablesDeclaration(CSSyntax.VariableDeclarationSyntax Decl, ITyp
                     equalsValueClauseSyntax = null;
                 } else {
                     var returnBlock = SyntaxFactory.Block(SyntaxFactory.ReturnStatement(adjustedInitializerExpr));
-                    _typeContext.PerScopeState.Hoist(new HoistedParameterlessFunction(GetInitialValueFunctionName(vbName), csTypeSyntax, returnBlock));
+                    _typeContext.PerScopeState.Hoist(new HoistedFunction(GetInitialValueFunctionName(vbName), csTypeSyntax, returnBlock, null));
```

Comment:
Agreed this null is slightly unpleasant. More so the field that ends up being null - which I generally wouldn't expect without being indicated by name/annotation (I don't mind having nullable enable at the top of some files, just a wholesale move to it is too much work). I'm surprised you didn't just create the empty parameterlist here.

Another option would have been to make it something like an IReadOnlyList of the parameters that gets passed and stored away (passing through the minimum information needed, and left it to the output methods (e.g. AsLocalFunction) to decide how to format it.

Feel free to tweak it further if either of those sound better to you, but it's a pretty minor issue

Reply:
> I'm surprised you didn't just create the empty parameterlist here.
I kind of forgot that's an option :)